fbu-app/ViewControllers/HomeScreens/ModifyHouseViewController.m

```diff
@@ -0,0 +1,127 @@
+//
+//  ModifyHouseViewController.m
+//  fbu-app
+//
+//  Created by sophiakaz on 7/25/19.
+//  Copyright © 2019 lucjia. All rights reserved.
+//
+
+#import "ModifyHouseViewController.h"
+#import "PlainRoomateCell.h"
+#import "Parse/Parse.h"
+#import "House.h"
+#import "Persona.h"
+
+@interface ModifyHouseViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@property (nonatomic, strong) NSMutableArray *housemates;
+@property (weak, nonatomic) IBOutlet UIButton *houseButton;
+@property (weak, nonatomic) IBOutlet UIButton *removeButton;
+- (IBAction)tapRemove:(id)sender;
+
+@end
+
+@implementation ModifyHouseViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    
+    [self reloadView];
+    // Do any additional setup after loading the view.
+    self.tableView.tableFooterView = [[UIView alloc]
+                                      initWithFrame:CGRectZero];
+}
+
+- (void)viewWillAppear:(BOOL)animated {
+    [super viewWillAppear:animated];
+    [self reloadView];
+}
+
+- (void) reloadView {
+    [self setButtonLabel];
+    [self fetchHousemates];
+    [self.tableView reloadData];
+}
+
+
+- (void) setButtonLabel {
+    Persona *persona = [[PFUser currentUser] objectForKey:@"persona"];
+    [persona fetchIfNeeded];
```

Comment:
Instead of using fetchIfNeeded use fetchIfNeededInBackgroundWithBlock: instead throughout the program in order to avoid "Warning: A long-running operation is being executed on the main thread".